packages/ibm-products/src/components/ExportModal/ExportModal.tsx

```diff
@@ -275,7 +284,8 @@ export let ExportModal = forwardRef(
           aria-label={title}
           size="sm"
           preventCloseOnClickOutside
-          {...{ open, ref, onClose, ...getDevtoolsProps(componentName) }}
+          onClose={closeModal}
+          {...{ open, ref, ...getDevtoolsProps(componentName) }}
```

Comment:
since we are using <ComposeModal, I could see a prop that will serve this purpose. Can you try this?
![image](https://github.com/user-attachments/assets/8676ed08-100c-4784-ad5a-6af1314484c5)